plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -395,6 +395,24 @@ public void testUnregisterTableAccessControl()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateTableInNonExistentSchemaFails()
+    {
+        String tableName = "test_create_table_in_non_existent_schema_" + randomNameSuffix();
+        String location = generateTableLocation(tableName);
+        assertQueryFails(
+                "CREATE TABLE non_existing_schema." + tableName + " (a int, b int) WITH (location = '" + location + "')",
```

Comment:
> non_existing_schema

This test runs on Glue (if you don't move to connector test). If `non_existing_schema schema` is accidentally created for some reason, it violates the test purpose. It would be better to add the random suffix.